packages/miniflare/test/plugins/email/index.spec.ts

```diff
@@ -51,7 +51,7 @@ test("Unbound send_email binding works", async (t) => {
 		compatibilityDate: "2025-03-17",
 	});
 
-	t.teardown(() => mf.dispose());
+	onTestFinished(() => mf.dispose());
```

Comment:
Should we use `disposeWithRetry()` everywhere rather than calling `mf.dispose()` directly?

Reply:
Or where it is being called within a `onTestFinished()` block, perhaps `useDispose()`?